miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppPermission.kt

```diff
@@ -1,5 +1,7 @@
 package com.rakuten.tech.mobile.miniapp.permission
 
+import androidx.annotation.Keep
+
 /** Type of miniapp permission. **/
 enum class MiniAppPermissionType(val type: String) {
```

Comment:
Does this one also need `@Keep`?

Reply:
Currently, there is no affect has been traced for this class without `@Keep`. But for safety we can address it here, what do you think @minh-rakuten?

Reply:
@khairul-alam-licon I guess just putting `@Keep` on data class `MiniAppCustomPermission` will help you fix the problem, no need to update `@Keep` for enum in `MiniAppPermission.kt`. I'm not sure so let try it :)

Reply:
@minh-rakuten I understand your concern, as `MiniAppCustomPermission` has a `pairValues`. Here, I re-checked that, adding `@Keep` annotation only in `MiniAppCustomPermissionType` and `MiniAppCustomPermissionResult` helps actually. So, I updated this PR. For now, let's keep it to understand clearly where we require this annotation.